gitpod-io/gitpod
# Remove useless escape in email regex

## Description

This removes a useless escape in the email regexp used to validate a user’s email address. The escaped double quotes character is part of the exclusion group that means that double quotes are forbidden in email addresses.

As [reported by eslint](https://eslint.org/docs/latest/rules/no-useless-escape), escaping a double quote in a regex string that’s in /literal/ notation is superfluous.

## Related Issue(s)

#3841 for the broader effort to reduce warnings

## How to test

(This behavior should be identical on `main` and this branch.)

1. In the navbar, click on your avatar
2. Click on **Settings**
3. In the settings UI, add a double quote character (`"`) to your email address
4. Press Enter
5. See that an error is shown
   ![image](https://user-images.githubusercontent.com/28510368/210882940-3fef1c79-9caa-4b92-99ad-7e3e290c3645.png)
6. Remove the character added in 3
7. See that the form can be saved
   ![image](https://user-images.githubusercontent.com/28510368/210883066-edf6645b-8099-4e2c-ab59-56fb7ac7df6e.png)

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
